platforms/core-configuration/kotlin-dsl-tooling-builders/src/main/kotlin/org/gradle/kotlin/dsl/tooling/builders/KotlinBuildScriptModelBuilder.kt

```diff
@@ -124,15 +141,18 @@ object KotlinBuildScriptModelBuilder : ToolingModelBuilder {
         parameter: KotlinBuildScriptModelParameter
     ): KotlinScriptTargetModelBuilder {
 
+        println("Y0: $modelRequestProject - $parameter")
```

Comment:
🤔 It might be interesting to keep the debug output via some `inline fun debugln(...)` function that operates based on a constant flag

Reply:
Oh my, I completely did not intend to leave it here 😆 
I wouldn't keep it for debug logging. It was just my attempts at getting specific assumptions verified in CrossVersion tests, where debugging is cumbersome

Reply:
Fixed